app/models/fields_definition.rb

```diff
@@ -52,14 +52,13 @@ def self.editable_by(user)
 
   alias_attribute :position, :pos
 
-  # this is tested in unit/account_test
```

Comment:
This comment is not even true anymore 😂 Since long ago it is tested in `test/unit/extra_fields_test.rb`